config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml

```diff
@@ -9,3 +9,27 @@ presubmits:
         command:
         - make
         - test
+  - name: pull-kube-storage-version-migrator-e2e
+    decorate: true
+    always_run: true
+    # TODO: remove this when the test is stable
+    optional: true
+    labels:
+      preset-service-account: "true"
```

Comment:
Where are these labels being used?

Reply:
see https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L437-L455, you need them :-)